devito/archinfo.py

```diff
@@ -32,18 +32,19 @@ def get_cpu_info():
 
     # Extract CPU flags and branch
     if lines:
-
         # The /proc/cpuinfo format doesn't follow a standard, and on some
         # more or less exotic combinations of OS and platform it might not
         # contain the information we look for, hence the proliferation of
         # try-except below
 
         def get_cpu_flags():
             try:
-                flags = [i for i in lines if i.startswith('flags')][0]
+                # ARM Thunder X2 is using 'Features' instead of 'flags'
+                flags = [i for i in lines if (i.startswith('Features')
+                                              or i.startswith('flags'))][0]
                 return flags.split(':')[1].strip().split()
             except:
-                return None
+                pass
```

Comment:
why this change? this will `return None` anyway, so it's clearer the other way

Reply:
I think I get your point. TO my understanding if the last pair of try-except in each function
`get_cpu_xxx`
is returning None, then we are fine.